providers/os/resources/docker.go

```diff
@@ -77,15 +83,30 @@ func (p *mqlDocker) containers() ([]interface{}, error) {
 			names = append(names, dContainer.Names[i])
 		}
 
-		/*
-			FIXME: ??? not used?
-			conn, err := connection.NewDockerEngineContainer(dContainer.ID)
-			if err != nil {
-				return nil, err
-			}
-		*/
+		localConnId++
+		containerConf := &inventory.Config{
+			Host: dContainer.ID,
+			Type: "docker-container",
+		}
+		containerAsset := &inventory.Asset{
+			Name:        dContainer.ID,
+			Connections: []*inventory.Config{containerConf},
+		}
+		containerConn, err := connection.NewDockerEngineContainer(localConnId, containerConf, containerAsset)
+		if err != nil {
+			return nil, err
+		}
+		containerAsset.Connections[0].Id = containerConn.ID()
+		containerRuntime := &plugin.Runtime{
+			Connection:     containerConn,
+			Callback:       p.MqlRuntime.Callback,
+			HasRecording:   p.MqlRuntime.HasRecording,
+			CreateResource: resources.CreateResource,
+			Upstream:       p.MqlRuntime.Upstream,
+		}
+		// s.runtimes[containerConn.ID()] = containerRuntime
```

Comment:
I think, the data is missing, because cnquery later uses the wrong connection to retrieve the data.